cli/dev/service.go

```diff
@@ -252,28 +285,21 @@ func ServiceLog(homeDirectory string, serviceName string, tailing bool) {
 		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, "'%v %v' %v", SERVICE_COMMAND, SERVICE_LOG_COMMAND, wderr)
 	}
 
-	// Get the deployment config. This is a top-level service because it's the one being launched, so it is treated as
-	// if it is managed by an agreement.
-	dc, _, cerr := serviceDef.ConvertToDeploymentDescription(true)
-	if cerr != nil {
-		cliutils.Fatal(cliutils.CLI_GENERAL_ERROR, "'%v %v' %v", SERVICE_COMMAND, SERVICE_LOG_COMMAND, cerr)
```

Comment:
Still need this code to locate the container once the service def is found. You can move this to after the searchDependencies

Reply:
Addressed this in recent changes